docs/about/release-notes.rst

```diff
@@ -46,6 +46,7 @@ Features
 * Added support for arbitrary unit, degrees Celsius and other special units `#2931 <https://github.com/scipp/scipp/pull/2931>`_.
 * :class:`scipp.Dataset` now supports ``drop_coords``, which returns :class:`scipp.Dataset` without the given coordinate by names  `#2940 <https://github.com/scipp/scipp/pull/2940>`_.
 * :class:`scipp.DataArray` now supports ``drop_coords``/ ``drop_masks``/ ``drop_attrs``, which returns :class:`scipp.DataArray` without the given coordinates/masks/attributes names by names `#2940 <https://github.com/scipp/scipp/pull/2940>`_.
+* :py:func:`scipp.DataGroup._repr_html_` now shows more structured information of :class:`scipp.DataGroup` instances in Jupyter notebooks.
```

Comment:
As `DataGroup` is completely new, we can skip adding individual changes to the release notes.